package.json

```diff
@@ -297,9 +297,9 @@
 		"test-unit:update": "npm run test-unit -- --updateSnapshot",
 		"test-unit:watch": "npm run test-unit -- --watch",
 		"pretest-unit-php": "wp-env start",
-		"test-unit-php": "wp-env run phpunit \"phpunit -c /var/www/html/wp-content/plugins/gutenberg/phpunit.xml.dist --verbose\"",
+		"test-unit-php": "wp-env run tests-wordpress /var/www/html/wp-content/plugins/gutenberg/vendor/bin/phpunit -c /var/www/html/wp-content/plugins/gutenberg/phpunit.xml.dist \"--verbose\"",
```

Comment:
The quotes around verbose -- do those need to be around all of the arguments passed in? Or are they making wp-env pass them into the container?

Reply:
I don't actually know why that was there, I've removed the quotes and it works the same as it did before.

Reply:
Ah, it must have been left over from when we had to wrap the entire command with quotation marks.